src/components/Profile/components/OfficeInfoList/UpdateOfficeLocationDialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import { FormControl, IconButton, Input, InputLabel } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import GordonDialogBox from 'components/GordonDialogBox';
+import GordonSnackbar from 'components/Snackbar';
+import { useState, useEffect } from 'react';
+import userService from 'services/user';
+import SearchField from 'views/PeopleSearch/components/SearchFieldList/components/SearchField';
+
+const UpdateOffice = () => {
+  const [open, setOpen] = useState(false);
+  const [room, setRoom] = useState('');
+  const [building, setBuilding] = useState('');
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  const [buildings, setBuildings] = useState([]);
+
+  const handleSubmit = async () => {
+    try {
+      await userService.updateOfficeLocation({ BuildingDescription: building, RoomNumber: room });
+      createSnackbar('Your office location will update within a couple hours.', 'success');
+    } catch {
+      createSnackbar('Office location failed to update. Please contact CTS.', 'error');
+    }
+    setOpen(false);
+  };
+
+  const createSnackbar = (message, severity) => {
+    setSnackbar({ message, severity, open: true });
+  };
```

Comment:
is there any specific benefit to having a helper function that simply adds an `open` tag

Reply:
answered by Evan in the edit office hours pull request